lib/shared/addon/components/cluster-driver/driver-linodelke/template.hbs

```diff
@@ -0,0 +1,343 @@
+                }}
+              {{/input-or-display}}
+            </div>
+          </div>
+        {{/if}}
+        <div class="row">
+          <div class="header mt-20 clearfix">
+            <div class="pull-left">
+              <h2 class="mb-0">
+                {{t "clusterNew.linodelke.tags.label"}}
+              </h2>
+            </div>
+          </div>
+          <div class="col span-5">
+            <label class="acc-label">
+              {{t "clusterNew.linodelke.tags.newTag"}}
+            </label>
+            {{#input-or-display editable=true value=newTag}}
+              {{input
+                type="text"
+                name="tags"
+                classNames="form-control"
+                placeholder=(t "clusterNew.linodelke.tags.placeholder")
+                value=newTag
+              }}
+            {{/input-or-display}}
+          </div>
+          <div class="col span-1">
+            <button class="btn bg-primary btn-sm mt-30" {{action "addNewTag"}}>
+              <i class="icon icon-plus"></i>
+            </button>
+          </div>
+          <div class="col span-6">
+            <div class="lke-tags">
+              {{#each cluster.lkeEngineConfig.tags as |tag tagIdx|}}
+                <!--     single tag start -->
+                <div class="lke-tag bg-primary">
+                  <span>
+                    {{tag}}
+                  </span>
+                  <button
+                    class="lke-delete bg-error"
+                    {{action "deleteTag" tagIdx}}
+                  >
+                    <i class="icon icon-trash"></i>
+                  </button>
+                </div>
+                <!--     single tag end  -->
+              {{else}}
+                No tags added
+              {{/each}}
+            </div>
+          </div>
+        </div>
+      {{/accordion-list-item}}
+      {{top-errors errors=errors}}
+      {{save-cancel
+        btnLabel="clusterNew.linodelke.clusterConfig.next"
+        savingLabel="clusterNew.linodelke.clusterConfig.loading"
+        save="verifyClusterConfig"
+        cancel=close
+      }}
+    {{else if (eq step 3)}}
+      {{#accordion-list-item
+        title=nodePoolConfigTitle
+        detail=nodePoolConfigDetail
+        expandAll=expandAll
+        expand=(action expandFn)
+        expandOnInit=true
+      }}
+        {{! select node pool }}
+        <div class="row">
+          <div class="col span-4">
+            <label class="acc-label">
+              {{t "clusterNew.linodelke.selectedNodePoolType.label"}}
+            </label>
+            {{#input-or-display editable=true value=selectedNodePoolType}}
+              {{new-select
+                class="form-control"
+                content=nodePoolChoises
+                value=selectedNodePoolType
+              }}
+            {{/input-or-display}}
+          </div>
+          <div class="col span-1">
+            <div class="acc-label pb-15">
+              Monthly:
```

Comment:
Missed bare string. Colon should not be moved to translation.

Can you also address the remaining labels that have bare strings in this file as well? Thanks!

Reply:
does GB or $ go to translations?

Reply:
We should have a Unit for "GB" yes. Unless you're doing full internationalization with the currency as well I would be okay creating a generic `usd: ${ amount }` translation for this part as well.

Reply:
I utilized the generic gigabyte, and circumvented the dollar by using the number formatter for currency